packages/react-next/jest.config.js

```diff
@@ -10,8 +10,8 @@ const config = createConfig({
     '@uifabric/react-hooks/lib/(.*)$': '@uifabric/react-hooks/lib-commonjs/$1',
     '@uifabric/utilities/lib/(.*)$': '@uifabric/utilities/lib-commonjs/$1',
     // These mappings allow Jest to run snapshot tests against Example files.
-    'office-ui-fabric-react/lib/(.*)$': '<rootDir>/../office-ui-fabric-react/src/$1',
-    'office-ui-fabric-react$': '<rootDir>/../office-ui-fabric-react/src/',
+    'office-ui-fabric-react/lib/(.*)$': 'office-ui-fabric-react/lib-commonjs/$1',
+    'office-ui-fabric-react$': 'office-ui-fabric-react/lib-commonjs/',
```

Comment:
@xugao's comment made me realize, actually what needs to be done here is this (and leave the comment):
```suggestion
    'office-ui-fabric-react/lib/(.*)$': 'office-ui-fabric-react/lib-commonjs/$1',
    'office-ui-fabric-react$': 'office-ui-fabric-react/lib-commonjs/',
    '@fluentui/react-next/lib/(.*)$': '<rootDir>/src/$1',
    '@fluentui/react-next$': '<rootDir>/src/',
```

Reply:
we currently don't have example tests in `react-next`. i don't mind adding this though